### DISCUSSION TITLE
Can I create my own Items and ActiveItem?

### DISCUSSION THREAD

**Discussion by nucombo** [canton7/Stylet#384] [Q&A]
How could I initialize and create my own Items and ActiveItem I tried to look it up but using say:
```cs
public ConductorBaseWithActiveItem<IScreen> MyOwnActiveItem {get;set;} 
```
Even thou it works it gives me some converters errors while using the default ActiveItem and Items I dont get any errors.

**Comment by canton7** [comment]
I'm afraid I don't understand your question. Can you please post the code you've tried, along with the errors that you see?
